sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -176,29 +176,6 @@ case class DescribeDatabase(
 }
 
 /**
- * A command that renames a table/view.
- *
- * The syntax of this command is:
- * {{{
- *    ALTER TABLE table1 RENAME TO table2;
- *    ALTER VIEW view1 RENAME TO view2;
- * }}}
- */
-case class AlterTableRename(
```

Comment:
moved to `tables.scala`. I just moved this one command for now to avoid inflating the diff too much.